modules/nf-core/mageck/count/tests/main.nf.test

```diff
@@ -0,0 +1,71 @@
+// nf-core modules test mageck/count
+nextflow_process {
+
+    name "Test Process MAGECK_COUNT"
+    script "../main.nf"
+    process "MAGECK_COUNT"
+
+    tag "modules"
+    tag "modules_nfcore"
+    tag "mageck"
+    tag "mageck/count"
+
+    test("mus_musculus - fastq") {
+
+       // config "./nextflow.config"
+
+        when {
+            process {
+                """
+                input[0] = [ [ id:'test,test2', single_end:true] , // meta map
+                [file(params.modules_testdata_base_path + 'crisprseq/testdata/SRR23101364_1.small.fastq.gz', checkIfExists: true),
```

Comment:
Now I get this:
  ERROR ~ No such file or directory:                                         │
│ https://raw.githubusercontent.com/nf-core/test-datasets/modules/data/crisprs │
│ eq/testdata/SRR23101364_1.small.fastq.gz

Maybe
```suggestion
                [file(params.modules_testdata_base_path + 'genomics/crisprseq/testdata/SRR23101364_1.small.fastq.gz', checkIfExists: true),
```
??

Reply:
ah nvm we need to add the files to the repo first 
https://github.com/nf-core/test-datasets/blob/modules/docs/ADD_NEW_DATA.md